websites/A/Animea/dist/metadata.json

```diff
@@ -0,0 +1,45 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "RisingSunLight",
+		"id": "240521747852558347"
+	},
+	"service": "Animea",
+	"description": {
+		"en": "Watch films in HD with language options, dubbing and subtiles.",
+		"fr": "Regardez des films en qualité HD avec des options de langue avec doublage et sous-titres."
```

Comment:
```suggestion
		"fr": "Regardez des films en HD avec des options de langues, doublage et sous-titres."
```

Reply:
Resolved in 401f95334db21f79fa5f8bbe9d999798d7fab508